src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -107,34 +106,33 @@
 
     $clr-datagrid-icon-size: rem(1);
 
-    .datagrid {
-        flex: 1 1 auto;
-        width: auto; // override the basic table's style of width: 100%
-        max-width: none; // override the basic table's style of max-width: 100%
+    .datagrid-overlay-wrapper {
+        display: flex;
+        flex-direction: row;
+        flex: 0 0 100%;
```

Comment:
If you don't mind, I'd rather have the whole flexbox setup separate and streamlined in the section that starts at line 13. It's complex enough that we should have styles split into layout on one side, and colors/backgrounds/borders/everything else on the other.